app/models/skill.rb

```diff
@@ -0,0 +1,6 @@
+class Skill < ActiveRecord::Base
+  SKILL_LEVELS = {"Beginner": 1, "Intermediate": 3, "Expert": 5, "Leader": 7}
+  validates_presence_of :name
+  validates_presence_of :level
```

Comment:
you should check out Rails enums

```Ruby
enum level: {
  beginner: 1,
  intermediate: 3,
  expert: 5,
  leader: 7
}
```
http://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html